x-pack/plugins/security_solution/server/endpoint/routes/actions/audit_log.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENDPOINT_ACTION_LOG_ROUTE } from '../../../../common/endpoint/constants';
+import { EndpointActionLogRequestSchema } from '../../../../common/endpoint/schema/actions';
+import { actionsLogRequestHandler } from './audit_log_handler';
+
+import { SecuritySolutionPluginRouter } from '../../../types';
+import { EndpointAppContext } from '../../types';
+
+/**
+ * Registers the Host-(un-)isolation routes
```

Comment:
Feels like this registration function shoudl be under `/actions` and register all of the routes for actions.

Unless, we see action log as not associated with "Isolation" (perhaps in the future we might have other activity shown)

Reply:
> we see action log as not associated with "Isolation" (perhaps in the future we might have other activity shown)

this is precisely the case

Reply:
I changed the copied comment bit as well. 2dd22ed92b3c54da9bdcce467dbe2d7e87097e4d